docs/Example-Queries.md

```diff
@@ -0,0 +1,122 @@
+# Example SQL queries for web analytics
+
+With Ahoy and [Blazer](https://github.com/ankane/blazer), you can create dashboards for your web analytics. Here are some sample queries to get you started.
+
+## Active Visitors
+
+Count of number of visitors over the last 1 day, 7 days, and 28 days.
+
+```sql
+SELECT
+    date,
+    daily AS "1 day",
+    SUM(daily) OVER (order by date rows between 7 preceding and current row) AS "7 day",
```

Comment:
Think this will overcount 7 day and 28 day uniques for people who visit on multiple days. Same with Active Users.

Reply:
Doh, good catch. I'm struggling to come up with a performant query (without subselects) that properly counts 7 and 28 day actives. If anyone has suggestions, it would be appreciated!